src/instafeed.js

```diff
@@ -227,6 +227,22 @@
     return transformedFiltered;
   };
 
+  Instafeed.prototype._getTags = function getTags(data) {
+    // This expression is any combination of letters and numbers, no punctuation.
+    // Strictly speaking, a hashtag can't consist only of numbers, but we don't enforce that here.
+    var exp = /#([A-zÀ-ÖØ-öø-ÿ0-9]+)/gi;
```

Comment:
Ahh — glad you included the accent character base. I initially hadn't thought about non-ascii hashtags.

However, now that I'm thinking about it, there are additional tags this character set wouldn't cover, for example non-latin based tags, and other unicode tags like emoji:

* #コーヒー (Japanese)
* #☕

Since the character set is potentially very large, I think we'll probably have to take a different approach to parsing the tags. Some options include:

* Using the "not-these-characters" `[^abc]` matcher, and checking for the special characters that aren't allowed
* Doing some looping over the string and checking each character.

Reply:
Yeah, this is a good point. I based this very roughly off of Twitter's hash-tag patterns: https://github.com/twitter/twitter-text/blob/master/js/src/regexp/validHashtag.js - which say that a valid hash-tag is any combination of Latin letters and numbers, as long as there's at least one number. It's not the exhaustive set, but I figured it was a reasonable start.

We could add a emoji ranges:

`(\u00a9|\u00ae|[\u2000-\u3300]|\ud83c[\ud000-\udfff]|\ud83d[\ud000-\udfff]|\ud83e[\ud000-\udfff])`

Japanese character ranges:

`[一-龯]`

But for further languages (Cyrillic, Koren, Ethiopian, etc. etc.) , we're better off with a `not-these` approach like you said.

I was definitely trying to avoid doing a lot of non-regex string processing: splitting stuff and examining each word/character sounds like a performance nightmare.

Reply:
Perhaps we could make the tag expression an option - the default could be latin characters + number + emoji, but if people wanted a bigger character set they could add their own?

Reply:
Switched to `#([\wÀ-úÀ-ÿ\u00a9|\u00ae|\u2000-\u3300|\ud83c\ud000-\udfff|\ud83d\ud000-\udfff|\ud83e\ud000-\udfff]+)` which covers Latin, emoji and Japanese katakana.